subsys/bluetooth/mesh/prov.c

```diff
@@ -58,8 +59,8 @@ int bt_mesh_prov_reset_state(void (*func)(const uint8_t key[64]))
 		bt_mesh_attention(NULL, 0);
 	}
 
-	memset(&bt_mesh_prov_link.dhkey, 0,
-	       sizeof(bt_mesh_prov_link) - offsetof(struct bt_mesh_prov_link, dhkey));
+	(void)memset((uint8_t *)&bt_mesh_prov_link + offset, 0,
+	       sizeof(bt_mesh_prov_link) - offset);
```

Comment:
It looks like (at least in the GitHub web UI) that the split line is not correctly aligned right after `memset(` on the first line.

Reply:
Done.